704_Binary_Search.py

```diff
@@ -0,0 +1,25 @@
+# n = len(nums)
+# time: O(log n)
+# space: O(1)
+
+from typing import List
+
+
+
+class Solution:
+    def search(self, nums: List[int], target: int) -> int:
+
+        right = len(nums) - 1      
+        left = 0
+        
+        if target not in nums:
```

Comment:
This line's time complexity is O(n).
So, it would be best if you rewrote it with time complexity O(log n).